docs/build-instructions/windows.md

```diff
@@ -33,7 +33,7 @@ cd atom
 script/build # Creates application in the `Program Files` directory
```

Comment:
My oversight in this instance. 
I'll open a new PR to correct this, as soon as we are also clear about whether further clarification needs to be added.